go.mod

```diff
@@ -20,15 +20,15 @@ require (
 	github.com/influxdata/influxdb-client-go/v2 v2.9.0
 	github.com/kelseyhightower/envconfig v1.4.0
 	github.com/tsenart/vegeta/v12 v12.8.4
-	go.opencensus.io v0.23.0
+	go.opencensus.io v0.24.0
```

Comment:
any reason why this was bumped?

Reply:
I assume it got bumped because the new version of grpc depends on this.
What I did was: 
1. bumping grpc to v1.56.3
2. run go mod tidy, then go mod vendor
3. and other things got updated as a result

Reply:
For sanity check I just checked out release-1.10 head and changed grpc and ran go mod tidy
```
-       go.opencensus.io v0.23.0
+       go.opencensus.io v0.24.0
        go.uber.org/atomic v1.9.0
        ....
        golang.org/x/time v0.1.0
-       google.golang.org/api v0.96.0
-       google.golang.org/grpc v1.48.0
+       google.golang.org/api v0.114.0
+       google.golang.org/grpc v1.56.3

```